lib/tilt/jbuilder.rb

```diff
@@ -103,9 +103,10 @@ def evaluate(scope, locals, &block)
     private
     def set_locals(locals, scope, context)
       view_path = options[:view_path]
-      scope.send(:instance_variable_set, '@_jbuilder_view_path', view_path)
-      scope.send(:instance_variable_set, '@_jbuilder_locals', locals)
-      scope.send(:instance_variable_set, '@_tilt_data', data)
+      scope.instance_exec(binding) {
+        @_jbuilder_view_path = view_path
+        @_jbuilder_locals = locals
+      }
```

Comment:
@anthonator I see your concern and totally agree with it.

But if add that `data` back as third line, it raises a `NoMethodError`. Which leads us to the initial issue: I don't understand where is defined. 😉